TypeScript compilation error when running pnpm build in core/ (Counter)

Hello,

I encountered an issue while running pnpm build in your project. The error message I received is:

```
lib/useCases/CounterInteractor.ts:20:28 - error TS2345: Argument of type 'number | undefined' is not assignable to parameter of type 'number'.
  Type 'undefined' is not assignable to type 'number'.

20     this.counter.increment(qty);
                              ~~~


lib/useCases/CounterInteractor.ts:30:28 - error TS2345: Argument of type 'number | undefined' is not assignable to parameter of type 'number'.
  Type 'undefined' is not assignable to type 'number'.

30     this.counter.decrement(qty);
                              ~~~
```

Steps to reproduce the issue:
Run pnpm install
Run pnpm build
Observe the TypeScript compilation error mentioned above.

Environment:
Operating System: Windows 11

--- a/counter/core/lib/entities/counter.ts
+++ b/counter/core/lib/entities/counter.ts
@@ -5,15 +5,11 @@ export class Counter {
     this.count = startNumber;
   }
 
-  increment(qty: number) {
-    if(qty !== null) {
-      this.count += qty;
-    }
+  increment(qty?: number) {
+    this.count += qty ? qty : 1;
   }
 
-  decrement(qty: number) {
-    if(qty !== null) {
-      this.count -= qty;
-    }
+  decrement(qty?: number) {
+    this.count -= qty ? qty : 1;
   }
 }

